frontends/web/src/decorators/load.tsx

```diff
@@ -100,7 +102,7 @@ export function load<LoadedProps extends ObjectButNotFunction, ProvidedProps ext
             private allEndpointsLoaded(): boolean {
                 if (this.endpoints === undefined) { return false; }
                 for (const key of Object.keys(this.endpoints) as KeysOf<LoadedProps>) {
-                    if ((this.state === null) || (this.state[key] === undefined)) {
+                    if (this.state[key] === undefined) {
```

Comment:
nice so it is back to the original https://github.com/digitalbitbox/bitbox-wallet-app/blob/d6b33a6d25e394e11bcb8055db41ff77129769ca/frontends/web/src/decorators/load.tsx#L103

👍